.github/workflows/tests.yml

```diff
@@ -21,6 +21,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
this is why it wasn't caught on the PR itself

Reply:
ah that's a good point, thanks for catching that